### ISSUE TITLE
t-clock straight up not working in 21H2 (build 19044.1149)

### ISSUE THREAD

**Issue by kusti420** [White-Tiger/T-Clock#263]

**Comment by miklosc** [comment]
I can say its not working in 21H1 19043.1151 either

**Comment by LynxusRen** [comment]
![image](https://user-images.githubusercontent.com/93513469/141409294-5a63dd70-d36d-46e3-b56d-1c28e61eb5d3.png)
Yes it is not working in 21h2 version

**Comment by CentralPier9** [comment]
Yep same issue here. Really sad that it's no longer working. I enjoyed it so much. It was working fine on my device on 21H2 but I did install it during 1903. After a recent issue forced me to reset my laptop, I tried downloading it but it was no longer working since after the restart it installed 21H2.

**Comment by Maki711** [comment]
It is still working on 21H1 (Build 19043.1706) and on 21H2 (Build 19044.1645).
I really wish somebody would pick this up and continue further the development...

**Comment by Walkman100** [comment]
just happened to see this as I was checking here for something else, I'm using Win10 21H2 b19044.1706, and T-Clock is working perfectly as normal...

**Comment by Maki711** [comment]
> just happened to see this as I was checking here for something else, I'm using Win10 21H2 b19044.1706, and T-Clock is working perfectly as normal...

~~but how?~~
~~the process does not even start on 21H2 19044.1645. Tried it on multiple PC with same Windows version and nothing.~~

~~what is the difference between 19044.1645 and 19044.1706 ?~~

Never mind, it magically started working on 19044.1645. No idea why.

**Comment by Walkman100** [comment]
For the record I've never had an issue with it not starting on Win10 since I started using it years ago, maybe your settings are corrupt? They're stored in the registry, annoyingly: #16

**Comment by kusti420** [comment]
> For the record I've never had an issue with it not starting on Win10 since I started using it years ago, maybe your settings are corrupt? They're stored in the registry, annoyingly: #16

ive had multiple clean installs in the time i made that post, it hasnt even worked straight after a clean install. dont think thats the problem
